sources/api/host-containers/src/main.rs

```diff
@@ -134,34 +120,23 @@ mod error {
 
 type Result<T> = std::result::Result<T, error::Error>;
 
-/// Query the API for the currently defined host containers
-async fn get_host_containers<P>(socket_path: P) -> Result<HashMap<Identifier, model::HostContainer>>
+/// Read the currently defined host containers from the config file
+fn get_host_containers<P>(config_path: P) -> Result<HashMap<Identifier, config::HostContainer>>
 where
     P: AsRef<Path>,
 {
-    debug!("Querying the API for settings");
-
-    let method = "GET";
-    let uri = constants::API_SETTINGS_URI;
-    let (code, response_body) = apiclient::raw_request(&socket_path, uri, method, None)
-        .await
-        .context(error::APIRequestSnafu { method, uri })?;
-    ensure!(
-        code.is_success(),
-        error::APIResponseSnafu {
-            method,
-            uri,
-            code,
-            response_body,
-        }
-    );
-
-    // Build a Settings struct from the response string
-    let settings: model::Settings =
-        serde_json::from_str(&response_body).context(error::ResponseJsonSnafu { method, uri })?;
+    debug!("Reading containers from the config file!");
+    let config_path = config_path.as_ref();
```

Comment:
More information is better when debugging.

```suggestion
    let config_path = config_path.as_ref();
    debug!("Reading containers from the config file: {}", config_path.display());
```